snmp/CHANGELOG.md

```diff
@@ -43,7 +43,7 @@
 
 ***Added***:
 
-* Update dependencies ([#18185](https://github.com/DataDog/integrations-core/pull/18185))
+* Update dependencies ([#18187](https://github.com/DataDog/integrations-core/pull/18185))
```

Comment:
shouldn't the link also be updated ?
```suggestion
* Update dependencies ([#18187](https://github.com/DataDog/integrations-core/pull/18187))
```

Reply:
@NouhaManai96 good catch, fixed!

My editor didn't highlight that somehow so I thought the link was fixed too.